model/src/main/java/io/syndesis/model/connection/Connector.java

```diff
@@ -33,6 +33,7 @@
 public interface Connector extends WithId<Connector>, WithName, WithProperties, Serializable {
 
     @Override
+    @Value.Default
```

Comment:
I removed `defaultAsDefault` Immutables `@Style` configuration on `ImmutablesStyle` as it was generating properties for helper methods (such as `isEndpointProperty`), so here we need to mark it as `@Default` to generate immutable property (Builder's `with...` method)